installation/index.md

```diff
@@ -36,13 +36,16 @@ The previously mentioned [openHABian](openhabian.html) can also be used to kicks
 
 ## Prerequisites
 
-Make sure that you have an up to date version of the [Oracle Java Runtime Environment](https://java.com/en/download/manual.jsp) on your host system installed.
-A famous choice is OpenJDK, however, Oracle Java Version 8 is recommended for openHAB.
-openHAB may be unstable and slower executed by OpenJDK.
+Make sure that you have an up to date Java Runtime Environment (JRE) installed on your host system. The [Oracle JRE](https://java.com/en/download/manual.jsp)
```

Comment:
I would actually by now change the recommendation to Zulu. The main reason is that there are "no license fees or field of use restrictions".
Oracle JDK is (at lease on embedded systems) alway subject to certain restrictions. You are actually only allowed to use it for development, but not for production. And a "pure" openHAB user probably does not really qualify as doing active development work. See https://blog.takipi.com/running-java-on-docker-youre-breaking-the-law/. Interesting is also that [Oracle does not even seem to care to comment such concerns](https://community.oracle.com/thread/3913114).